src/components/AttachmentView.js

```diff
@@ -44,24 +46,32 @@ const defaultProps = {
 };
 
 const AttachmentView = (props) => {
-    // Check both sourceURL and file.name since PDFs dragged into the the text field
-    // will appear with a sourceURL that is a blob
-    if (Str.isPDF(props.sourceURL)
+    // Check both source and file.name since PDFs dragged into the the text field
+    // will appear with a source that is a blob
+
+    // handles SVG
+    if (_.isFunction(props.source)) {
+        return (
+            <Icon src={props.source} height={variables.defaultAvatarPreviewSize} width={variables.defaultAvatarPreviewSize} />
+        );
+    }
+
+    if (Str.isPDF(props.source)
```

Comment:
Looks like the comment was supposed to follow the PDF check :D I also capitalized the `h` in `handles` and suggested a new comment for the SVG rendering

```suggestion
    // Handles case where source is a component (ex: SVG)
    if (_.isFunction(props.source)) {
        return (
            <Icon src={props.source} height={variables.defaultAvatarPreviewSize} width={variables.defaultAvatarPreviewSize} />
        );
    }

    // Check both source and file.name since PDFs dragged into the the text field
    // will appear with a source that is a blob
    if (Str.isPDF(props.source)
```